src/page/index.html

```diff
@@ -3,16 +3,23 @@
 <!-- {{@config.VERSION}} -->
 <html id="wire-main-app">
   <head>
-#include('meta.htm')
-#include('graph.htm')
+    #include('meta.htm') #include('graph.htm')
     <title>{{@config.BRAND_NAME}}</title>
     <style type="text/css">
       #include('../../../server/dist/static/style/default.css');
     </style>
   </head>
 
   <body>
-    #include('wire-main.htm')
+    <app-container
+      id="wire-main"
+      class="off"
+      data-bind="css: mainClasses()"
+      params="root: $root"
+      data-uie-name="status-webapp"
+      data-uie-value="is-loading"
```

Comment:
I think those need to be move inside the react component. 
It needs to be dynamic